ui/src/views/Quickstart/index.jsx

```diff
@@ -47,11 +46,11 @@ const initialYaml = {
       metadata: {
         name: '',
         description: '',
-        owner: '{{ event.head.user.email }}',
-        source: '{{ event.head.repo.url }}',
+        owner: '{{ event.sender.login }}@users.noreply.github.com',
```

Comment:
In json-e, we don't use `{{ }}` syntax. It would be `${ }`

It's possible that `${ }` characters need to be escaped

Reply:
If I change it to `${}`
it needs event variable. I don't think its available, am I missing something here?

Reply:
`${}` is part of json-e syntax, not javascript's template literal. That's what I mean when I say these need to be escaped. Its meaning is the same as meaning of `{{ }}` syntax. Actually, I think I was wrong about escaping `${}` - should be fine in a string, just like `{{ }}`